spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectMessageListenerContainer.java

```diff
@@ -212,18 +206,6 @@ public void addQueueNames(String... queueNames) {
 		super.addQueueNames(queueNames);
 	}
 
-	@Override
-	public void addQueues(Queue... queues) {
-		try {
-			addQueues(Arrays.stream(queues)
```

Comment:
OK! After closer look I really see your logic and what you suggest.
Not enough coffee? Or to many snow outside 😄 

Yes, I think we can go ahead with your suggestion and even really make those `Queue`-based methods as `final`.

Thank you for such a comprehensive analysis and patience to my nit-pickings.